rotkehlchen/globaldb/handler.py

```diff
@@ -268,6 +284,7 @@ def retrieve_assets(filter_query: 'AssetsFilterQuery') -> Tuple[List[Dict[str, A
                         'chain': ChainID.deserialize_from_db(entry[12]).serialize(),
                         'token_kind': EvmTokenKind.deserialize_from_db(entry[13]).serialize(),
                         'decimals': entry[3],
+                        'underlying_tokens': underlying_tokens.get(entry[0], None),
```

Comment:
by default get returns None if the key is not present but if you want to be explicit here is fine